PWGLF/Tasks/hyperRecoTask.cxx

```diff
@@ -43,16 +43,21 @@ namespace
 constexpr double betheBlochDefault[1][6]{{-1.e32, -1.e32, -1.e32, -1.e32, -1.e32, -1.e32}};
 static const std::vector<std::string> betheBlochParNames{"p0", "p1", "p2", "p3", "p4", "resolution"};
 static const std::vector<std::string> particleNames{"He3"};
+std::shared_ptr<TH1> hEvents;
+std::shared_ptr<TH1> hZvtx;
+std::shared_ptr<TH2> hNsigma3HeSel;
+std::shared_ptr<TH2> hDeDx3HeSel;
+std::shared_ptr<TH2> hDeDxTot;
 } // namespace
 
 namespace o2::aod
 {
 namespace hyperrec
 {
 DECLARE_SOA_COLUMN(IsMatter, isMatter, bool);      // bool: true for matter
-DECLARE_SOA_COLUMN(Px, px, float);                 // Momentum of the candidate (x direction)
```

Comment:
Hi @njacazio, could you tell us what we would improve by putting the table definition in another file and folder?
For us, it will duplicate the number of files we work with.

Reply:
Hi @mpuccio it's an improvement for the PWG code organization to avoid future proliferation of same data models. If you cannot do it now I can help you, unless you have other reasons for not splitting the data model.

Reply:
Btw I sincerely don't get the problem of working with an additional header file, this is how we always did since aliroot

Reply:
As you say, another improvement of the new analysis model is modularity and with this approach it is somehow negated. Therefore, while this may not seem directly useful right now, as far as I can tell, the modularity is actually highly desirable. Plus, it is the baseline rationale behind the framework https://aliceo2group.github.io/analysis-framework/docs/gettingstarted/theo2physicsrepo.html

Reply:
But again, when and if this table is required by some other task I can understand the move to a common place. But before that, this just reduces our flexibility.

Reply:
There cannot be any other user task that benefits from the existence of this code if the table definitions are not moved to the data model first. This is why we have this separation in our repository: to enhance collaboration within our PWG

Reply:
Exactly! And we don't want to have other tasks depending on this table because:
a) we are still developing it, so sudden change can happen
b) we don't want to have implicit dependencies (someone starts using it, and then we have to take care not to break anything)
We regularly present at the PAG, so if we want to have a common skim across different groups that is the place to start this conversation, in my opinion.

Reply:
It is even more important to organize the code in the standard way right from the start before it's too late. Plus, as you know, it is the whole analysis software that is under development. Being under development is not an argument to avoid that data-model tables get used by others. You can consider putting a disclaimer in the code for that. Plus, by allowing openness and profiting from others using your data model, you would gain essential feedback in spotting issues, thus improving the code's robustness. 
In addition, the standard structure makes it clear and transparent for everyone, making it much easier to follow changes and overall development.
@mpuccio

Reply:
```
Being under development is not an argument to avoid that data-model tables get used by others.
```
If the software is being developed and does not have a stable API (and the table are interfaces to tasks), you don't release it for the users.
But still, the code is open! People can see it here. The `DataModel` folder should be used for shared code, and this is not shared code for the time being. Take the modifications in this PR: we moved to a completely different meaning of the variables that could have silently broken code using it.
What I am saying is very simple: for now, this code is used only for a single task, and people know we have it. If they are interested, they can tell us, and we find a way to have a sustainable development model. 
Simply putting the code into the `DataModel` folder does not enable robustness and people looking at the code.
Adding the code to the `DataModel` folder with a disclaimer on top saying "Before using this send an email to..." does not really help, as it just makes our life more complicated and does not enable collaboration more than having the code here.
As I said, @njacazio, if you or anyone else knows of people interested in using our code, you know how to find us.